plugins/capistrano/capistrano.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+# Added `shipit` because `cap` is a reserved word. `cap` completion doesn't work.
```

Comment:
So, I *think* that means:
* rename `_capistrano` to `_shipit`
* in `shipit()`, change `cap $*` to `command cap $*` (not `noalias cap $*`, because that could pick up the `zsh/cap` builtin if the module has been loaded)
* tell the user to `alias cap=shipit`, or just do it for them in `capistrano.plugin.zsh`

I'd have to try it out to be sure; it's tricky. Maybe people would be okay with using `capi` for capistrano, instead of `cap`, since `cap` is already taken by the shell? But then again, `cap` is already taken by capistrano, too. Not sure what the Right Thing to do here is.